src/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -1287,6 +1293,7 @@ public static void ReadTooPreciseDouble()
                 Assert.Throws<FormatException>(() => root.GetUInt64());
 
                 Assert.Throws<InvalidOperationException>(() => root.GetString());
+                Assert.Throws<InvalidOperationException>(() => root.GetBytesFromBase64());
```

Comment:
About half of the "invalid state" test blocks are missing `TryGetBytesFromBase64`